js/event_target_test.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, assert, assertEqual } from "./test_util";
+
+if (!Event) {
+  let Event = function() {};
```

Comment:
I'm not sure what the right solution here is, but I don't think this will work as expected. This `Event` binding will only be in scope for the duration of the consequent block since `let` is block scoped. Maybe this is what you should do instead?

```suggestion
   var Event = function() {};
```

Reply:
`var`s get hoisted to the top of the scope, so it would be self-defeating if you used `var` (you would never see the global `Event`).  I didn't catch this.  You would need to do checking on `window.Event` or `globalThis.Event`... something like:

```ts
const Event = globalThis.Event || function () {};
```

But this will _always_ run in Deno's runtime and therefore `Event` should always be defined.